packages/headless/src/api/search/search-api-params.ts

```diff
@@ -114,17 +114,28 @@ export interface ExcerptLength {
   excerptLength?: number;
 }
 
+export interface AuthenticationParam {
+  authentication?: string;
```

Comment:
The reason is that the actual parameter expected by the API is a string, joined with commas for each provider. 

Since in this file were trying to represent as closely as possible what the API expect, I went with the approach using the "exact type".